charts/pravega/templates/pravega.yaml

```diff
@@ -91,6 +91,15 @@ spec:
       repository: {{ .Values.image.repository }}
       pullPolicy: {{ .Values.image.pullPolicy }}
     controllerReplicas: {{ .Values.controller.replicas }}
+    {{- if .Values.controller.svcNameSuffix }}
```

Comment:
Is it possible to set default value if no value is provided in values file?

Reply:
I can do that, but if someone tries to install pravega using this charts and old pravega operator openAPI schema will throw error as this field is not present in CRD of previous operator.

Reply:
okay, let's keep it as is for now then. Thanks